link

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env php
+<?php
+
+/*
+* This file is part of the Symfony package.
+*
+* (c) Fabien Potencier <fabien@symfony.com>
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+/**
+ * Link dependencies to components to a local clone of the monolithic Symfony repository.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+
+if ($argc < 2) {
+    echo 'Link dependencies to components to a local clone of the monolithic Symfony repository.'.PHP_EOL.PHP_EOL;
+    echo 'Usage: ./link /path/to/the/project'.PHP_EOL;
+    exit(1);
+}
+
+if (!is_dir("$argv[1]/vendor/symfony")) {
+    echo "The directory \"$argv[1]\" does not exist or the dependencies are not installed, did you forget to run \"composer install\" in your project?";
+    exit(1);
+}
+
+$sfPackages = array();
+foreach (glob(__DIR__.'/src/Symfony/{Bundle,Bridge,Component}/*', GLOB_BRACE | GLOB_ONLYDIR | GLOB_NOSORT) as $dir) {
+    $sfPackages[json_decode(file_get_contents("$dir/composer.json"))->name] = $dir;
+}
+
+foreach (glob("$argv[1]/vendor/symfony/*", GLOB_BRACE | GLOB_ONLYDIR | GLOB_NOSORT) as $dir) {
+    if (is_link($dir)) {
+        echo "\"symfony/$package\" is already a symlink, skipping.".PHP_EOL;
+        continue;
+    }
+
+    $package = basename($dir);
+    if (isset($sfPackages["symfony/$package"])) {
+        rename($dir, "$dir.back");
+        symlink($sfDir = $sfPackages["symfony/$package"], $dir);
+        echo "\"symfony/$package\" has been linked to \"$sfDir\".";
```

Comment:
same.. missing EOL. Output is terrible ;-)

Reply:
I update (I didn't noticed because I use ZSH 😛).